frontend/src/BookOverview.tsx

```diff
@@ -27,15 +25,11 @@ function BookOverview() {
         setModalIsOpen(false)
     }
 
-/*    if (books === undefined) {
-        return <>Bitte haben Sie einen Augenblick Geduld...</>
-    }*/
     return <>
-        <CreateBookModal modalIsOpen={modalIsOpen} closeModal={closeModal} reloadAllBooks={fetchAllBooks}/>
+        <CreateBook modalIsOpen={modalIsOpen} closeModal={closeModal} reloadAllBooks={fetchAllBooks}/>
         {books.length > 0 ?
 
             <>
-                <div>
                     <table>
```

Comment:
hm maybe you can here use https://www.w3schools.com/html/html5_semantic_elements.asp elements and style them :)